code/regen_lib.py

```diff
@@ -165,6 +165,10 @@ def load_bib(stream: io.StringIO):
     def customize(record):
         record = bibx.customization.convert_to_unicode(record)
 
+        # Replace slashes with the U+2215 division symbol in 'title' field
+        if 'title' in record:
+            record['title'] = record['title'].replace('/', '\u2215')
+
```

Comment:
I would not add it here. While it's a smart idea, the file names are generated by `get_file_title_from_entry`, a few lines below. Instead of just returning `f"{entry.title}.md"` we can apply this conversion there:
```python
def get_file_title_from_entry(entry: Entry) -> str:
	title = entry.title.replace('/', '-')
    return f"{title}.md"
```
This makes *only* the filenames to be affected by this change, instead of *everything* (since you change the `Entry` itself). While most platforms use Unicode nowadays, I'm afraid of using it due to Windows and legacy filesystems that might not like non-ascii characters in filenames.

Reply:
Agree on everything. Thanks for the review!